source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,20 +1,103 @@
 
-Example request: `https://local-authority-eng.register.gov.uk/register`
+### Return all entries since your last update
 
-Download a full new copy of the register by using [the `GET /download-register` endpoint](#get-download-register).
+You can use the [`GET /entries/` endpoint](#getentries) with the `start` and `limit` parameters to return specific entries to a register. 
 
-Download all updates for one record according to a particular key by [using the `GET /records/{key}/entries` endpoint](#get-records-key-entries):
+For example, the last highest entry number you got from the `country` register could be 205. You could then use the following request to find out if there are any new entries:
 
-Example URL: `https://local-authority-eng.register.gov.uk/records/KIN/entries`
+Example request:
 
-Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/records/KIN/entries --header 'Accept: application/json' --header 'Authorization: YOUR-API-KEY-HERE'`
+```http
+GET /entries?start=206/ HTTP/1.1
+Host: country.register.gov.uk
+Accept: application/json
+Authorization: YOUR-API-KEY-HERE
+```
+
+Example response:
+
+```http
+HTTP/1.1 200
+Content-Type: application/json
+Link: <?start=206>; rel="next"
+
+[
+  {
+    "index-entry-number": "206",
+    "entry-number": "206",
+    "entry-timestamp": "2017-03-29T14:22:30Z",
+    "key": "GM",
+    "item-hash": [
+      "sha-256:0429375c4fb403288ef816e5dd38a24f192e35b8f55e40cc6266eb25eaef77b1"
+    ]
+  },
+  {
+    "index-entry-number": "207",
+    "entry-number": "207",
+    "entry-timestamp": "2017-10-25T09:52:52Z",
+    "key": "CI",
+    "item-hash": [
+      "sha-256:b3ca21b3b3a795ab9cd1d10f3d447947328406984f8a461b43d9b74b58cccfe8"
+    ]
+  },
+  {
+    "index-entry-number": "208",
+    "entry-number": "208",
+    "entry-timestamp": "2018-06-13T13:54:40Z",
+    "key": "SZ",
+    "item-hash": [
+      "sha-256:f89f36ed8b2a1417237a8e95b810e8ab4ead844277ad7bc7794cb5f83732c976"
+    ]
+  }
+]
+```
+
+To see what data is in the updates for each of these entries, you can use the [`GET /items/{item-hash}` endpoint](#items) on the value in the `item-hash` property. 
+
+Following the same example, you could use the following request for entry 208:
+
+Example request:
+
+```http
+GET /items/sha-256:f89f36ed8b2a1417237a8e95b810e8ab4ead844277ad7bc7794cb5f83732c976
+Host: country.register.gov.uk
+Accept: application/json
+Authorization: YOUR-API-KEY-HERE
+```
 
 Example response:
 
+```http
+{
+  "country": "SZ",
+  "official-name": "Kingdom of Eswatini",
+  "name": "Eswatini",
+  "citizen-names": "Swazi"
+}
 ```
+
+### Return all updates for a particular key 
+
+You can use the [`GET /records/{key}/entries/` endpoint](#get-record-key-entries) to download all updates for a record. 
```

Comment:
Describing these as "updates" is perhaps misleading here. This endpoint returns all entries that exist for that key since the beginning of time. They are therefore not updates in the sense that you could get something you already have, unlike the approach in the section above, where you will always be returned entries you don't already have.

It doesn't help that although this endpoint returns entries in the same way that the `/entries` endpoint does, it does not support pagination. This endpoint also currently has a limit of returning 5000 entries. So if there are more than 5000 entries for a key, this request will fail. It might be worth adding that there is a limit in the docs for now. Then we should fix it and consider adding pagination. 

Thoughts @arnau?

Reply:
Also, should we use "get" rather than "download"?

Reply:
Mmm, I wonder if we should consider adding pagination to `/records/{key}/entries/`. It is a collection that can grow big after all.